src/kibana/components/index_patterns/_cast_mapping_type.js

```diff
@@ -21,6 +21,7 @@ define(function (require) {
         { name: 'geo_shape',    type: 'geo_shape',  group: 'geo'    },
         { name: 'ip',           type: 'ip',         group: 'other'  },
         { name: 'attachment',   type: 'attachment', group: 'other'  },
+        { name: 'murmur3',      type: 'murmur3',    group: 'hash'  },
```

Comment:
Won't cause problems, but maybe we should remove the comma from the end of this line?